docs/source/cli/graph-construction/distributed/gsprocessing/aws-infra/emr.rst

```diff
@@ -11,9 +11,10 @@ we suggest going through its
 to familiarize yourself with its concepts.
 
 In summary, we will launch an EMR cluster configured in a way that will allow
-us to run jobs with executors that use the GSProcessing EMR Docker image,
-and then launch our job using ``spark-submit`` from the
-cluster's leader node.
+us to run jobs with executors that use the GSProcessing EMR Docker image, and then launch our job through
+[EMR Step](https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-spark-submit-step.html). For advanced users who
```

Comment:
Can we mention the EMR step script here?

Reply:
I think it is already mention here? The whole doc here is for EMR step script.

Reply:
I mean https://github.com/awslabs/graphstorm/blob/main/graphstorm-processing/scripts/submit_gsp_emr_step.py do we link to this script somewhere?

Reply:
Yes. It is here: https://github.com/awslabs/graphstorm/pull/998/files#diff-e9816a7488329c63cc094f4bc72d558a653c08eb54001618b963d8467de78e9dR104